assets/src/dashboard/app/views/templateDetails/index.js

```diff
@@ -208,7 +208,7 @@ function TemplateDetails() {
               <DetailViewNavBar
                 ctaText={__('Use template', 'web-stories')}
                 handleBookmarkClick={
-                  enableBookmarks && handleBookmarkClickSelected
+                  enableBookmarks ? handleBookmarkClickSelected : null
```

Comment:
Small update so we don't get proptype warnings about `handleBookmarkClick` expecting a function but received a boolean when `enableBookmarks` is `false`.